src/tr/pijamalikoi/src/eu/kanade/tachiyomi/extension/tr/pijamalikoi/PijamaliKoi.kt

```diff
@@ -0,0 +1,35 @@
+package eu.kanade.tachiyomi.extension.tr.pijamalikoi
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import okhttp3.Request
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class PijamaliKoi : MangaThemesia(
+    "Pijamalı Koi",
+    "https://pijamalikoi.com/m",
+    "tr",
+    mangaUrlDirectory = "/seri",
+    dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale("tr")),
+) {
+    override fun chapterListRequest(manga: SManga): Request {
+        return super.chapterListRequest(manga).fixupUrl()
+    }
+
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return super.mangaDetailsRequest(manga).fixupUrl()
+    }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        return super.pageListRequest(chapter).fixupUrl()
+    }
+
+    // Fixes the `/m` in baseUrl
+    private fun Request.fixupUrl(): Request {
+        return takeIf { url.pathSize >= 1 }
```

Comment:
when does it remove the /m/ ? The site seems to just have it for manga and chapter urls

Reply:
If everything is working as expected, it will remove it every time. There's something in `*Request()` that makes the URL look like `https://example.com/m/m/seri/regina-rena-to-the-unforgiven/`.

On second thought, I guess it would fail only if the URL is something like `https://example.com`, but at that point you have bigger problems.

I'll change it so it removes the extra segment unconditionally.